src/plugins/location-finder/Box.vue

```diff
@@ -25,17 +25,11 @@
       ButtonClick() {
         console.log( this.location )
         if(this.location.id && (this.location.indoors == undefined)) {
-          let fileUrl = GameService.getLocationImgPath(this.location);
-          console.log(fileUrl);
-
-          let fileLink = document.createElement('a');
-          fileLink.href = fileUrl;
-          fileLink.setAttribute('download', this.location.id+ '.jpg');
-
-          document.body.appendChild(fileLink)
-          fileLink.click();
-          document.body.removeChild(fileLink);
+          this.$refs.LocationImageDownloadLink.click()
         }
+      },
+      DownloadLink() {
+        return GameService.getLocationImgPath(this.location);
```

Comment:
Not a blocker, but you might want to change that to be something coming from "computed" instead of in "methods". Note: This would mean that the template would be `:href="DownloadLink"` (instead of calling it)